apps/site/lib/site_web/templates/trip_plan/_to_from_inputs.html.eex

```diff
@@ -30,15 +31,18 @@
     <div id="trip-plan__search--from__results" class="c-search-bar__results"></div>
     <div id="trip-plan__location-error--from" role="alert" aria-live="assertive" tabindex="0" class="c-trip-plan-widget__location-error location-error"><%= @from_error %></div>
   </div>
-
+  <div id="trip-plan-reverse-control" class="c-trip-plan-widget__reverse-control-vert">
```

Comment:
sorry didn't realize this earlier but instead of needing to the rotation via css you can just use different icons that are already right/left oriented.

https://fontawesome.com/search?q=long-arrow&s=solid%2Cbrands

Reply:
Went with the rotation since getting the arrows side by side was already present